helpers/new-package.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Creates a new package based on templates
+# Author: Zachary Loeber
+
+HEIGHT=15
+WIDTH=80
+BINPATH=${BINPATH:-"tmp/build.helpers"}
+
+APP=$(whiptail --inputbox "Application Name" 8 78 "newapp" --title "Application Info" 3>&1 1>&2 2>&3)
+if [ $? -ne 0 ]; then
+    exit 0
+fi
+
+if [ -d "vendor/${APP}" ]; then
+    echo  "vendor/${APP} already exists!"
+    exit 1
+fi
+
+OPTIONS=(bin "github binary"
+         tarball "github tarball"
+         custom_tarball "custom tarball")
+
+packageType=$(whiptail \
+    --clear \
+    --title "New Package Form" \
+    --menu "Package Template Type" \
+    $HEIGHT $WIDTH 4 \
+    "${OPTIONS[@]}" \
+    3>&1 1>&2 2>&3)
+
+if [ $? -ne 0 ]; then
+    exit 0
+fi
+
+VENDOR=$(whiptail --inputbox "Appcation Vendor" 8 78 "vendor" --title "Application Info" 3>&1 1>&2 2>&3)
+if [ $? -ne 0 ]; then
+    exit 0
+fi
+
+DESC=$(whiptail --inputbox "Appcation Description" 8 78 "A short description" --title "Application Info" 3>&1 1>&2 2>&3)
+if [ $? -ne 0 ]; then
+    exit 0
+fi
+
+VERSION=$(whiptail --inputbox "Appcation Version" 8 78 "0.0.0" --title "Application Info" 3>&1 1>&2 2>&3)
```

Comment:
We just pull this from github. See https://github.com/cloudposse/packages/blob/master/tasks/Makefile.package#L101